SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -178,14 +178,15 @@ bool FLocalDeploymentManager::KillProcessBlockingPort(int32 Port)
 	return bSuccess;
 }
 
-bool FLocalDeploymentManager::LocalDeploymentPreRunChecks()
+bool FLocalDeploymentManager::LocalDeploymentPreRunChecks(const uint16_t& RuntimeGRPCPort)
 {
 	bool bSuccess = true;
 
 	// Check for the known runtime ports which could be blocked by other processes.
-	TArray<int32> RequiredRuntimePorts = { RequiredRuntimePort, WorkerPort, HTTPPort, SpatialGDKServicesConstants::RuntimeGRPCPort };
 
-	for (int32 RuntimePort : RequiredRuntimePorts)
+	TArray<uint16_t> RequiredRuntimePorts = { RequiredRuntimePort, WorkerPort, HTTPPort, RuntimeGRPCPort};
+	
+	for (uint16_t RuntimePort : RequiredRuntimePorts)
```

Comment:
Perhaps for more clarity, we change `RuntimePort` to `RequiredPort`?

Reply:
Sure, that makes sense